task-definition-b2-backup.tftpl

```diff
@@ -0,0 +1,108 @@
+[
+  {
+    "dnsSearchDomains": null,
+    "logConfiguration": {
+      "logDriver": "awslogs",
+      "options": {
+        "awslogs-group": "${cw_log_group}",
+        "awslogs-region": "${aws_region}",
+        "awslogs-stream-prefix": "${cw_stream_prefix}"
+      }
+    },
+    "entryPoint": null,
+    "portMappings": [],
+    "command": [],
+    "linuxParameters": null,
+    "cpu": ${cpu},
+    "environment":
+      {
+        "name": "ATLAS_TOKEN",
```

Comment:
Change to `TFE_TOKEN`?

Reply:
`ATLAS_TOKEN` is required by `tfc-ops` which is used by `tfc-dump.pl`.

Reply:
That is correct. I somehow forgot you were using tfc-ops